### ISSUE TITLE
Add keypoints for iris detection

### ISSUE THREAD

**Issue by abdou31** [tzutalin/dlib-android-app#52]
Hello , 
I want to add other keypoints to detect iris how can i do that?
Shoul i train the model?
Thanks

**Comment by aboozaid** [comment]
Yeah you should train your own CNN model but I don't know if there's a way to use it inside dlib otherwise you should use TensorFlow instead.
